tests/scripts/thread-cert/test_ping.py

```diff
@@ -197,19 +197,6 @@ def verify(self, pv: pktverify.packet_verifier.PacketVerifier):
             .filter_ping_request() \
             .must_next()
 
-        # TODO: Enable this section
```

Comment:
Removed since it's unclear whether these checks are necessary: https://github.com/openthread/openthread/issues/6494